plugins/woo-ai/src/shared/error-message.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Define the Property types for the RandomLoadingMessage component
+interface ErrorMessageProps {
```

Comment:
It's a bit more common for us to use `type` than interface in the mono-repo, but totally optional.